src/coreclr/jit/compiler.cpp

```diff
@@ -5154,6 +5154,8 @@ void Compiler::placeLoopAlignInstructions()
     {
         // Adding align instruction in prolog is not supported
         // hence just remove that loop from our list.
+        fgFirstBB->bbFlags &= ~BBF_LOOP_ALIGN;
```

Comment:
This is a case where you don't call `unmarkLoopAlign`?

Reply:
Yes, probably I should use that.